.github/workflows/spectral-validation.yml

```diff
@@ -3,10 +3,10 @@ name: Run Spectral Validation on all samples
 on:
   pull_request:
     branches: [ "main" ]
-    paths: [ "calm/draft/**", "calm/pattern/**" ]
+    paths: [ "calm/draft/**", "calm/pattern/**", "calm/samples/**", "spectral/**" ]
   push:
     branches: [ "main" ]
-    paths: [ "calm/draft/**", "calm/pattern/**" ]
+    paths: [ "calm/draft/**", "calm/pattern/**", "calm/samples/**", "spectral/**" ]
 
 permissions:
```

Comment:
Do we still need the explicit permissions, think this was just an attempt to get the packaged action to work.